package.json

```diff
@@ -0,0 +1,71 @@
+{
+  "name": "pokemon-api-backend",
+  "version": "0.0.1",
+  "description": "",
+  "author": "",
+  "private": true,
+  "license": "UNLICENSED",
+  "scripts": {
+    "build": "nest build",
+    "format": "prettier --write \"src/**/*.ts\" \"test/**/*.ts\"",
+    "start": "nest start",
+    "start:dev": "nest start --watch",
+    "start:debug": "nest start --debug --watch",
+    "start:prod": "node dist/main",
+    "lint": "eslint \"{src,apps,libs,test}/**/*.ts\" --fix",
+    "test": "jest",
+    "test:watch": "jest --watch",
+    "test:cov": "jest --coverage",
+    "test:debug": "node --inspect-brk -r tsconfig-paths/register -r ts-node/register node_modules/.bin/jest --runInBand",
+    "test:e2e": "jest --config ./test/jest-e2e.json"
+  },
+  "dependencies": {
+    "@nestjs/common": "9.0.0",
+    "@nestjs/core": "9.0.0",
+    "@nestjs/mongoose": "9.2.1",
+    "@nestjs/platform-express": "9.0.0",
```

Comment:
We will use express instead fastify?

Reply:
Yes! once the structure is "done" migrate to fastify

Reply:
Okok